src/components/bottomSheet/BottomSheet.tsx

```diff
@@ -602,8 +619,42 @@ const BottomSheetComponent = forwardRef<BottomSheet, BottomSheetProps>(
         if (_providedOnChange) {
           _providedOnChange(index);
         }
+
+        AccessibilityInfo.isScreenReaderEnabled().then(isEnabled => {
```

Comment:
lets keep this logic out of the library and let user customise it as they pleased with `onChange` prop.

Reply:
I've removed the "AccessibilityInfo.isScreenReaderEnabled"-logic out of the library.